lib/routes/ln_address/ln_address_fee_message.dart

```diff
@@ -0,0 +1,116 @@
+            boxPadding: const EdgeInsets.fromLTRB(16, 30, 16, 16),
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Text(
+                  _formatFeeMessage(context, openingFeeParams!),
+                  style: themeData.textTheme.titleLarge,
+                  textAlign: TextAlign.center,
+                ),
+              ],
+            ),
+          )
+        : const SizedBox();
+  }
+
+  String _formatFeeMessage(BuildContext context, OpeningFeeParams openingFeeParams) {
+    final texts = context.texts();
+    final currencyState = context.read<CurrencyBloc>().state;
+    final accountState = context.read<AccountBloc>().state;
+    final paymentOptionsState = context.read<PaymentOptionsBloc>().state;
+    // Proportional value is ppm(parts per million)
+    final proportionalPercent = (openingFeeParams.proportional / (1000000 / 100)).toString();
+    final liquiditySats = accountState.maxInboundLiquidity;
+    final liquidityFormatted = currencyState.bitcoinCurrency.format(liquiditySats);
+    final liquidityAboveMin = liquiditySats > 1;
+
+    // Calculate the maximum receivable amount within the fee limit (in millisatoshis)
+    final channelFeeLimitSat = paymentOptionsState.channelFeeLimitMsat ~/ 1000;
+    // Calculate the maximum sendable amount (in sats)
+    final maxSendableSats = maxReceivableSat(accountState, channelFeeLimitSat, openingFeeParams);
+    final maxSendableFormatted = currencyState.bitcoinCurrency.format(maxSendableSats);
+
+    // Get the minimum sendable amount (in sats), can not be less than 1 or more than maxSendable
+    final minSendableSats = (liquidityAboveMin) ? 1 : openingFeeParams.minMsat ~/ 1000;
+    final minSendableAboveMin = minSendableSats > 1;
+    final minSendableFormatted = currencyState.bitcoinCurrency.format(minSendableSats);
+    if (!minSendableAboveMin) {
+      return "Minimum sendable amount can't be less than ${currencyState.bitcoinCurrency.format(1)}.";
+    }
+    if (minSendableSats > maxSendableSats) {
+      return "Minimum sendable amount can't be greater than maximum sendable amount.";
+    }
+
+    final isFeeApplicable = maxSendableSats < liquiditySats;
+    if (!isFeeApplicable) {
+      // Send more than {minSendableSats} and up to {maxSendableSats} to this address.
+      return texts.invoice_ln_address_channel_not_needed(
+        minSendableFormatted,
+        maxSendableFormatted,
+      );
+    } else if (liquidityAboveMin) {
+      // Send more than {minSendableSats} and up to {maxSendableSats} to this address. A setup fee of {proportionalPercent}% with a minimum of {minSendableSats}
+      // will be applied for sending more than {liquiditySats}.
+      return texts.invoice_ln_address_warning_with_min_fee_account_connected(
+        minSendableFormatted,
+        maxSendableFormatted,
+        proportionalPercent,
+        minSendableFormatted,
+        liquidityFormatted,
+      );
```

Comment:
In this case there needs to be a `minFeeFormatted` (from openingFeeParams.minMsat), as minSendable will be 1 but the setup minimum also needs displaying
```suggestion
      // Send more than {minSendableSats} and up to {maxSendableSats} to this address. A setup fee of {proportionalPercent}% with a minimum of {minFeeSats}
      // will be applied for sending more than {liquiditySats}.
      return texts.invoice_ln_address_warning_with_min_fee_account_connected(
        minSendableFormatted,
        maxSendableFormatted,
        proportionalPercent,
        minFeeFormatted,
        liquidityFormatted,
      );
```

Reply:
Is this applicable to other messages as well?

Reply:
I think in that case minSendableSats equals the openingFeeParams.minMsat, so it's ok. You could use minFeeFormatted just for clarity

Reply:
Right, keeping it as it is for now.